src/include/time_inline.h

```diff
@@ -199,8 +199,14 @@ __wt_op_timer_fired(WT_SESSION_IMPL *session)
 {
     uint64_t diff, now;
 
-    if (!F_ISSET(session->txn, WT_TXN_RUNNING) || session->operation_start_us == 0 ||
-      session->operation_timeout_us == 0)
+    /*
+     * Internal threads and operations happening outside of a transaction are usually excluded from
+     * the operation timeout mechanism. Bypass that for pre-fetch until the timeout API decision has
+     * been revisited and/or a better solution is found.
```

Comment:
Is there a WT ticket you could link to?

Reply:
@korteland There is the original ticket which changed the timeout API, but I think it would make more sense if I created a new ticket to reconsider this design decision.

Reply:
OK, sounds good. I think we should add a reference to that new ticket in the code so it's clear what needs to change.

Reply:
The new ticket will be here: [WT-12905](https://jira.mongodb.org/browse/WT-12905)